services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ProtectedPathSpec.groovy

```diff
@@ -909,6 +909,80 @@ class ProtectedPathSpec extends BaseSpecification {
         database.resetCosts()
     }
 
+    def "Able to synchronize rules for a flow with protected path"() {
```

Comment:
I think this more belongs to 'rules synchronize' tests, because its main point is to verify the 'switch sync' handle and not protected path itself. SwitchRulesSpec maybe?

Reply:
will do it when pr https://github.com/telstra/open-kilda/pull/2265 is merged

Reply:
done